docs/source/user/extensions.rst

```diff
@@ -432,3 +466,28 @@ themes
 
 The ``themes`` directory contains assets (such as CSS and icons) for
 JupyterLab theme extensions.
+
+
+JupyterLab User Settings Directory
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The user settings directory contains the user-level settings for Jupyter extensions.
```

Comment:
There is extra whitespace after each sentence in this paragraph. Not a big deal, but anomalous.